### PR TITLE
1035 add spotify embed block

### PR DESCRIPTION
**Pull request by jalopez13**
Added Spotify Embed Block

### CODE REVIEW

File: src/plugins/blocksmith/components/spotify-embed-block-preview/index.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      frameBorder="0"
+      allow="encrypted-media"
+      title="Spotify Embed Block"
+    />
+  );
+
+  switch (spotifyType) {
+    case 'show':
+    case 'episode':
+      console.log('podcast');
```
Comment (gdbrown) [inline src/plugins/blocksmith/components/spotify-embed-block-preview/index.jsx]: @jalopez13 remove the debugging console.logs

**Comment by cohencodes** [comment]
@jalopez13 @gdbrown It does not look like we are using the parser util that Eric mentioned in the ticket: https://github.com/wb-apps/tmz-cms/issues/1035

Also, in `spotify-embed-block` you cannot type into the text area or edit any text after pasting into it

File: src/plugins/blocksmith/components/spotify-embed-block-preview/index.jsx (hunk 1 above)
Comment (jalopez13) [reply src/plugins/blocksmith/components/spotify-embed-block-preview/index.jsx]: @gdbrown 

Removed console.logs

**Comment by jalopez13** [comment]
@cohencodes 

We are not using parser. I added test cases for coverage.

**Pull request merged**